ddtrace/utils/formats.py

```diff
@@ -3,7 +3,7 @@
 from .deprecation import deprecation
```

Comment:
This change actually doesn't need to be included in this PR. I'll remove it for a separate one.